charts/nri-kube-events/README.md.gotmpl

```diff
@@ -0,0 +1,117 @@
+[![Community Plus header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Plus.png)](https://opensource.newrelic.com/oss-category/#community-plus)
```

Comment:
I left all comments in the templated file, nothing to check here, no?

Reply:
The template is generated based on this template. I have to make changes here instead of the README but it is ok. There is no need to duplicate review comments.